core/controller/src/main/resources/application.conf

```diff
@@ -2,6 +2,13 @@
 include "logging"
 include "akka-http-version"
 
+whisk {
+  loadbalancer {
+    invoker-busy-threshold: 4
```

Comment:
apropos my comment above about documentation - how does one override these values where today that's done in ansible vars.

Reply:
You still override them via ENV variables as today. Ansible is just a way of providing those environment variables.